controllers/cluster_controller_test.go

```diff
@@ -21,8 +21,10 @@ import (
 	"testing"
 
 	"github.com/gogo/protobuf/proto"
```

Comment:
If breaking these up, then "github.com/gogo/progobuf/proto" should be grouped with the imports below.

Reply:
move to below